php.ini-production

```diff
@@ -1764,20 +1764,20 @@ ldap.max_links = -1
 
 [opcache]
 ; Determines if Zend OPCache is enabled
-;opcache.enable=0
+;opcache.enable=1
 
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
-;opcache.enable_cli=0
+;opcache.enable_cli=1
```

Comment:
OPCache only caches for the duration of the current process, that means that the cache doesn't have any effect on CLI usually, if it does, then a negative effect. But OPCache doesn't just cache, it also does a lot of optimizations, and they are what matters to long running applications.